integrationTests/p2p/peerDisconnecting/peerDisconnecting_test.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var durationBootstrapingTime = time.Duration(time.Second * 2)
+var durationBootstrapingTime = 2 * time.Second
```

Comment:
Instead of []string{chooseNonCircuitAddress(advertiser.Addresses())} on line 45, you could use []string{integrationTests.GetConnectableAddress(advertiser)}, and remove chooseNonCircuitAddress func  from this file.

Reply:
done